CHANGES.rst

```diff
@@ -14,7 +14,8 @@ It is currently undergoing fast development and backward compatibility is not en
 
 Major changes
 -------------
-
+* The :class:`SentenceEncoder` is now available to encode columns with diverse entries.
```

Comment:
I think it's worth expanding the whatsnew entry as this is a rather high-profile addition. I would say that it allows representing entries in a table with embeddings computed by a deep-learning model